mllib/src/main/scala/org/apache/spark/ml/clustering/KMeans.scala

```diff
@@ -248,6 +269,11 @@ class KMeans @Since("1.5.0") (
       .setMaxIterations($(maxIter))
       .setSeed($(seed))
       .setEpsilon($(tol))
+
+    if (isSet(initialModel)) {
+      algo.setInitialModel($(initialModel).parentModel)
+    }
+
     val parentModel = algo.run(rdd)
     val model = new KMeansModel(uid, parentModel)
     copyValues(model)
```

Comment:
The final returned model will have param of initialModel. Can this model be saved and loaded without any issue?